tests/fixtures/migrations/hsl.php.expect

```diff
@@ -0,0 +1,127 @@
+<?hh // strict
+
+use namespace HH\Lib\{Str, Dict, Math, C};
+
+function hsl_migration_main(): void {
+
+  echo "\nucwords: ".Str\capitalize_words('hello world');
+  echo "\nucfirst: ".Str\capitalize("hello");
+  echo "\nstrtolower: ".Str\lowercase("HELLO");
+  echo "\nstrtoupper: ".Str\uppercase("hello");
+  echo "\nstr_replace: ".Str\replace('foobarbaz', 'foo', 'bar');
+  echo "\nstr_replace: ".
+    Str\replace_every('a test example with arrays', Dict\associate(['a', 'b'], ['x', 'y']));
+  $count = 0;
+  // this case is not handled in HSL
+  echo "\nstr_replace: ".str_replace('foo', 'bar', 'foobarbaz', &$count);
```

Comment:
Is this because of the `&count` ? If so, can you put that in the comment

Reply:
Yes, commented.